spec/financial_spec.rb

```diff
@@ -81,6 +81,12 @@
 
       expect(results).to eq(70.14566694692749)
     end
+
+    it 'raises an exception when the payment amount is lower than the interest amount' do
+      raise_message = "The payment amount ($ 100) is too low. It should be at least $ 560.0 (interest amount)"
+
+      expect { Exonio.nper(rate, 100, -pv, fv, 1) }.to raise_error(ArgumentError, raise_message)
```

Comment:
We should maintain the same Excel behavior: 

![screen shot 2018-10-10 at 11 52 06](https://user-images.githubusercontent.com/13101/46745252-1763d200-cc83-11e8-8f56-3634756f9654.png)

Reply:
@rizidoro I think that @cleicar tried to generate a scenario for a set of parameters that will cause an error in Excel. For example:

<img width="371" alt="screen shot 2018-10-12 at 8 43 03 am" src="https://user-images.githubusercontent.com/17584/46869800-e73f3f00-cdfa-11e8-8315-e00d7b9abb0d.png">

(payment amount is lower than interest amount)

If I slightly vary the payment amount (to be higher than the monthly interest amount) it works fine: 

<img width="442" alt="screen shot 2018-10-12 at 8 43 50 am" src="https://user-images.githubusercontent.com/17584/46869822-f58d5b00-cdfa-11e8-9bcf-cb26b43bdf08.png">

(payment amount is higher than interest amount)